lib/actions/deleteLinkFromProjectAction.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert');
+const DeleteLinkFromProjectResponse = require('./deleteLinkFromProjectResponse');
+const ProjectLink = require('../projectLink');
+
+class DeleteLinkFromProjectAction {
+  constructor(projectLink = ProjectLink) {
+    this.projectLink = projectLink;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  parseParams(userInputString = '') {
+    assert(typeof userInputString === 'string');
+    const parsedUserInput = userInputString.split(' ');
+
+    if (parsedUserInput.length < 2) {
+      throw new TypeError('invalid input');
+    }
+    const linkParams = { label: parsedUserInput[1], link: parsedUserInput[2] };
+    return linkParams;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  async execute(params, context) {
+    const paramsObj = this.parseParams(params);
+    // eslint-disable-next-line new-cap
+    const projectLink = new this.projectLink(context.project, paramsObj.label, paramsObj.link);
```

Comment:
This is probably working, but it isn't really doing what we want it to do. Currently, this line is creating a brand new instance of a new link object from the parameters provided by the user. However, we want to find an existing projectLink record and create an instance from that record. That means we should have some kind of find method to find a project link based on the project ID and the label. Then once we have that instance, we can use the delete method on that instance. Also, using a find method will allow us to change our response to the user, if the label they provided doesn't match any links in the database.